.github/actions/setup-eks-cluster/action.yml

```diff
@@ -33,11 +33,22 @@ runs:
            owner: "${{ inputs.owner }}"
 
         managedNodeGroups:
-        - name: ng-1
+        - name: ng-amd64
           instanceTypes:
            - t3.medium
-           - t3a.medium
-          desiredCapacity: 2
+          desiredCapacity: 1
+          spot: true
+          privateNetworking: true
+          volumeType: "gp3"
+          volumeSize: 10
+          taints:
+           - key: "node.cilium.io/agent-not-ready"
+             value: "true"
+             effect: "NoExecute"
+        - name: ng-arm64
+          instanceTypes:
+           - t4g.medium
```

Comment:
Are these instances available in enough capacity in your experience? I've had bad experiences on my end with arm64 hosts availability, but maybe it changed since then...

Reply:
I honestly haven't used them a ton, but I've not really heard of too many issues with capacity. I suspect it's a lot better than it used to be. We could also add other instance ARM types if it becomes a problem. This is just the cheapest option.